src/main/java/net/earthcomputer/clientcommands/command/ClientCommandHelper.java

```diff
@@ -77,8 +81,19 @@ public static Component getCommandTextComponent(@Translatable String translation
 
     public static Component getCommandTextComponent(MutableComponent component, String command) {
         return component.withStyle(style -> style.applyFormat(ChatFormatting.UNDERLINE)
-                .withClickEvent(new ClickEvent(ClickEvent.Action.RUN_COMMAND, command))
-                .withHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, Component.literal(command))));
+            .withClickEvent(new ClickEvent(ClickEvent.Action.RUN_COMMAND, command))
+            .withHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, Component.literal(command))));
     }
 
+    public static final Map<String, Runnable> runnables = new HashMap<>();
+
+    public static String registerCode(Runnable code) {
+        String randomString = new Random().ints(48, 122 + 1)
+            .filter(i -> (i <= 57 || i >= 65) && (i <= 90 || i >= 97))
```

Comment:
These can be char literals rather than magic constants. Although a better idea is to pick a random number between 0 and 26 + 26 + 10 and convert that to a lower- or uppercase letter or number with an if-else chain.